src/calculateRentalCost.js

```diff
@@ -4,7 +4,16 @@
  * @return {number}
  */
 function calculateRentalCost(days) {
-  // write code here
+  const rentPerDay = 40;
+  const advanceDiscount = 50;
+  const regularDiscount = 20;
+  const res = rentPerDay * days;
+
+  if (days > 2) {
+    return days > 6 ? res - advanceDiscount : res - regularDiscount;
+  }
+
+  return res;
```

Comment:
Consider using `const` for the `res` variable because its value is not being reassigned. This adheres to the checklist item that prefers `const` over `let` where possible.